src/components/Events/PreviewEvent.js

```diff
@@ -56,5 +84,15 @@ export default function PreviewEvent(props){
   );
 }
 
+const mapStateToProps = state => {
```

Comment:
no need to connect to redux;  newEvent info is already being passed down as props (props.newEvent.scheduleOptions and props.newEvent.restaurantOptions)